NEWS.md

```diff
@@ -22,3 +20,8 @@
 
 #### Posit Workbench
 -
+
+### Updated Dependencies
+- Updated GWT to version 2.10.0 (#11505; Desktop + Server)
```

Comment:
I chose to "tag" each update with the products which have these associated updates. We could also consider something like:

```
#### Common

#### Desktop Only

#### Server Only
```

But that seems fairly verbose, and since most dependencies are common between Desktop + Server it seems simpler to just tag each update as in this PR.

Reply:
Yeah - that is a bit much. I wouldn't include Common, Desktop Only, Server Only.